<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HNPrintInfoDialog</class>
 <widget class="QDialog" name="HNPrintInfoDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>497</width>
    <height>210</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <widget class="QPushButton" name="btnCancel">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>150</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnPrint">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>150</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leTitle">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>20</y>
     <width>361</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>海能仪器K1160全自动凯氏定氮仪实验报告</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leHeader">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>60</y>
     <width>361</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="leFooter">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>100</y>
     <width>361</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
